scripts/circleci-jobs/openshift3-integration-tests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+
+DOCKERHUB_USER = os.getenv('DOCKERHUB_USER')
+DOCKERHUB_PASSWORD = os.getenv('DOCKERHUB_PASSWORD')
+PROCESS = subprocess.run(["docker", "login", "--username", DOCKERHUB_USER, "--password", DOCKERHUB_PASSWORD], capture_output=True, text=True)
+print(PROCESS.stdout)
+
+CIRCLE_BRANCH = os.getenv("CIRCLE_BRANCH")
+if CIRCLE_BRANCH == "staging":
+    IMAGE_TAG = "staging-candidate"
+else:
+    IMAGE_TAG = "discardable"
```

Comment:
we're running os3 integration tests for regular branches as well? not just staging? I don't think we do that for all non-KinD integration tests, not sure we want to start doing it

Reply:
Agreed but this is a bit of a KinD-based integration test which allows us to run it on PRs. The other platform test suffer from requiring exclusivity on a single cluster, which would slow down PRs if opened in parallel.

Reply:
hmmm okay, let's start with your proposal, but be open minded towards pushing it to run only on merges if at some point we decide it's too much hassle to run on every branch?